packages/core-mobile/app/services/wallet/WalletService.tsx

```diff
@@ -649,10 +674,114 @@ class WalletService {
       this.walletType
     )
     const provXP = await NetworkService.getAvalancheProviderXP(
-      Boolean(network.isTestnet)
+      Boolean(network.isTestnet),
+      isDevnet(network)
     )
     return wallet.getReadOnlyAvaSigner({ accountIndex, provXP })
   }
+
+  public async createDummyImportPTx({
+    stakingAmount,
+    accountIndex,
+    avaxXPNetwork,
+    sourceChain,
+    destinationAddress,
+    feeState
+  }: CreateImportPTxParams & { stakingAmount: bigint }): Promise<UnsignedTx> {
+    const readOnlySigner = await this.getReadOnlyAvaSigner(
+      accountIndex,
+      avaxXPNetwork
+    )
+    const assetId = isDevnet(avaxXPNetwork)
+      ? DEVNET_AVAX_ASSET_ID
+      : avaxXPNetwork.isTestnet
+      ? TESTNET_AVAX_ASSET_ID
+      : MAINNET_AVAX_ASSET_ID
+
+    const utxos = getTransferOutputUtxos(
+      stakingAmount,
+      assetId,
+      destinationAddress ?? '',
+      DUMMY_UTXO_ID
+    )
+    const utxoSet = new utils.UtxoSet([utxos])
+    return readOnlySigner.importP({
+      utxoSet,
+      sourceChain,
+      toAddress: destinationAddress,
+      feeState
+    })
+  }
+
+  public async createDummyExportPTx({
+    amountInNAvax,
+    accountIndex,
+    avaxXPNetwork,
+    destinationChain,
+    destinationAddress,
+    feeState
+  }: CreateExportPTxParams): Promise<UnsignedTx> {
+    const readOnlySigner = await this.getReadOnlyAvaSigner(
+      accountIndex,
+      avaxXPNetwork
+    )
+    const provider = await NetworkService.getAvalancheProviderXP(
+      !!avaxXPNetwork.isTestnet,
+      isDevnet(avaxXPNetwork)
+    )
+    const assetId = provider.getAvaxID()
+    const utxos = getTransferOutputUtxos(
+      amountInNAvax * 2n,
```

Comment:
simply setting the utxo amount to be twice more to guarrantee the tx is constructed successfully and return the txFee.

Reply:
can you add this as a comment instead so it doesn't get lost?